helm/nessie/templates/deployment.yaml

```diff
@@ -142,28 +142,27 @@ spec:
             {{- end }}
             {{- end }}
 
-            {{- if .Values.jaegerTracing.enabled }}
-            {{- if .Values.jaegerTracing.serviceName }}
-            - name: QUARKUS_JAEGER_SERVICE_NAME
-              value: {{ .Values.jaegerTracing.serviceName }}
-            {{- end }}
-            {{- if .Values.jaegerTracing.publishMetrics }}
-            - name: QUARKUS_JAEGER_METRICS_ENABLED
-              value: {{ .Values.jaegerTracing.publishMetrics | quote }}
-            {{- end }}
-            {{- if .Values.jaegerTracing.samplerType }}
-            - name: QUARKUS_JAEGER_SAMPLER_TYPE
-              value: {{ .Values.jaegerTracing.samplerType }}
-            {{- end }}
-            {{- if .Values.jaegerTracing.samplerParam }}
-            - name: QUARKUS_JAEGER_SAMPLER_PARAM
-              value: {{ .Values.jaegerTracing.samplerParam | quote }}
-            {{- end }}
-            {{- if .Values.jaegerTracing.endpoint }}
-            - name: QUARKUS_JAEGER_ENDPOINT
-              value: {{ .Values.jaegerTracing.endpoint }}
+            {{- if .Values.tracing.enabled }}
+            - name: QUARKUS_OPENTELEMETRY_TRACER_ENABLED
+              value: "true"
+            - name: QUARKUS_OPENTELEMETRY_TRACER_EXPORTER_OTLP_ENDPOINT
+              value: {{ .Values.tracing.endpoint | quote }}
+            - name: QUARKUS_OPENTELEMETRY_TRACER_RESOURCE_ATTRIBUTES
+            {{- if .Values.tracing.attributes }}
+              value: "{{- include "nessie.dictToString" .Values.tracing.attributes }}"
+            {{- else }}
+              value: "service.name={{ include "nessie.name" . }}"
+            {{- end }}
+            {{- if .Values.tracing.sampler }}
+            - name: QUARKUS_OPENTELEMETRY_TRACER_SAMPLER
+              value: {{ .Values.tracing.sampler | toString | replace "true" "on" | replace "false" "off" | quote }}
```

Comment:
The calls to `replace` are required because in Yaml, `on` and `off` are actually booleans, but the server expects the strings `on` or `off`, not `true` or  `false`.